app/src/main/java/com/teamagam/gimelgimel/app/map/esri/EsriGGMapView.java

```diff
@@ -429,17 +428,10 @@ private void setupLocationDisplayer() {
     mLocationDisplayer.start();
   }
 
-  private Geometry transformToEsri(
-      com.teamagam.gimelgimel.domain.map.entities.geometries.Geometry geometry) {
+  private Geometry transformToEsri(com.teamagam.gimelgimel.domain.map.entities.geometries.Geometry geometry) {
```

Comment:
Not part of the task, but what about static import for our objects, so we won't use the full name of the class?